docs/sources/whatsnew/whats-new-in-v7-4.md

```diff
@@ -81,7 +81,7 @@ The main use case is for [multi-dimensional](https://grafana.com/docs/grafana/la
 
 You can now provide detailed information to alert notification recipients by injecting alert query data into an alert notification. Labels that exist from the evaluation of the alert query can be used in the alert rule name and in the alert notification message fields. The alert label data is injected into the notification fields when the alert is in the alerting state. When there are multiple unique values for the same label, the values are comma-separated.
 
-![Variable support in alert notifications](/img/docs/v74/alert-variable-notifications.png)
+![Variable support in alert notifications](/img/docs/alerting/notification_template.png)
```

Comment:
@wbrowne, from the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md):

Image file names should be descriptive and unique. Also, add the software version number that the image applies to or the screenshot was taken in. Example: share-dashboard-link-7-3.png.

If you can append the version number, that will be great!